mgr.go

```diff
@@ -76,13 +76,14 @@ type mountInfo struct {
 }
 
 type containerInfo struct {
-	state        containerState
-	rootfs       string
-	mntPrepRev   []mntPrepRevInfo
-	mounts       []mountInfo
-	uidInfo      uidInfo
-	shiftfsMarks []configs.ShiftfsMount
-	autoRemove   bool
+	state         containerState
+	rootfs        string
+	mntPrepRev    []mntPrepRevInfo
+	mntInfos      []mountInfo
```

Comment:
nit: suggest renaming "mntInfos" -> "reqMntInfos", to make it clear that these are the info for mounts requested by sysbox-runc.

Reply:
Done.